Lib/logging/__init__.py

```diff
@@ -1890,9 +1892,21 @@ def __init__(self, logger, extra=None):
         following example:
 
         adapter = LoggerAdapter(someLogger, dict(p1=v1, p2="v2"))
+
+        By default, LoggerAdapter objects will drop the "extra" argument
+        passed on the individual log calls to use its own instead.
+
+        Initializing it with merge_extras=True will instead merge both
+        maps when logging, the indivicual call extra taking precedence
+        over the LoggerAdapter instance extra
+
+        .. versionchanged:: 3.13
+           Added the ``merge_extras`` parameter.
         """
         self.logger = logger
         self.extra = extra
+        if merge_extras is not None:
```

Comment:
Don't need the condition, can always set `self.merge_extra = merge_extra`, right?